lms/djangoapps/django_comment_client/utils.py

```diff
@@ -878,6 +878,10 @@ def available_division_schemes(course_key):
     return available_schemes
 
 
+def enrollment_track_group_count(course_key):
```

Comment:
If you are going to introduce this, you might as well use it in the other places within this file where the number of enrollment tracks is computed.

Probably should add documentation too as it is a public method.